### PR TITLE
dev-to-alpha

### PR DESCRIPTION
**Pull request by njuettner**
* **Allow pods to direct traffic to itself via a service**
   <sup>Merge pull request #1470 from zalando-incubator/flannel-hairpin</sup>
* **Restore full sync behaviour for ExternalDNS**
   <sup>Merge pull request #1448 from zalando-incubator/upsert-off</sup>
* **Change admission controller and enable it fully**
   <sup>Merge pull request #1465 from zalando-incubator/enable-admission-controller-fully</sup>
* **Update emergency access service**
   <sup>Merge pull request #1469 from zalando-incubator/update-emergency-access-service</sup>
* **Update README.rst**
   <sup>Merge pull request #1467 from zalando-incubator/szuecs-patch-1</sup>
* **Update autoscaler to v1.3.1-teapot18**
   <sup>Merge pull request #1468 from zalando-incubator/autoscaler-taint-fix</sup>
* **Open a subset of ports on all workers for NLB support**
   <sup>Merge pull request #1462 from zalando-incubator/tracing-nlb</sup>
* **Upgrade zmon-agent**
   <sup>Merge pull request #1464 from zalando-incubator/upgrade-zmon-agent</sup>
* **Add teapot admission controller config**
   <sup>Merge pull request #1463 from zalando-incubator/enable-admission-controller</sup>
* **Deploy admission controller**
   <sup>Merge pull request #1459 from zalando-incubator/deploy-admission-controller</sup>
* **Rolling back Kubernetes Log Watcher**
   <sup>Merge pull request #1458 from zalando-incubator/rollback-logging-agent</sup>
* **Allow passing environments and strict label support for kubernetes-log-watcher**
   <sup>Merge pull request #1451 from zalando-incubator/new-log-watcher-release</sup>
* **Update auth webhook to v0.4.4**
   <sup>Merge pull request #1453 from zalando-incubator/update-webhook2</sup>
* **Change standard sc instead **WARNING(No message)
   <sup>Merge pull request #1454 from zalando-incubator/change-standard-sc-instead</sup>
* **Default storage class: explicitly specify zones**
   <sup>Merge pull request #1450 from zalando-incubator/explicit-zones</sup>
* **fix egress controller getting stuck create/update**
   <sup>Merge pull request #1449 from zalando-incubator/fix/egress-controller-stuck</sup>
* **Run external-dns with limited privileges**
   <sup>Merge pull request #1413 from zalando-incubator/secure-ext-dns</sup>

### CODE REVIEW

**Comment by njuettner** [comment]
@linki what is the status of external-dns v.0.5.8? Can we go on merging to alpha

**Comment by linki** [comment]
@njuettner Works as expected according to my tests. 👍 from me.

**Comment by njuettner** [comment]
👍

**Comment by mikkeloscar** [comment]
:+1:

File: cluster/cluster.yaml [hunk 1]
```diff
@@ -266,6 +266,11 @@ Resources:
           FromPort: 22
           IpProtocol: tcp
           ToPort: 22
+        # Allow the OpenTracing NLB to target these ports on all workers.
+        - CidrIp: 0.0.0.0/0
+          FromPort: 30443
+          IpProtocol: tcp
+          ToPort: 30444
```
Comment (linki) [inline cluster/cluster.yaml:273]: @vetinari this will add support for the lightstep NLB.

Comment (aermakov-zalando) [reply cluster/cluster.yaml:273]: By the way, is there any particular reason why is this not parameterised? Do we intend to deploy it to all clusters?

**Comment by linki** [comment]
👎  we need to double-check with torch

**Comment by mikkeloscar** [comment]
I put `do-not-merge` back as the opening of the NLB port would trigger violations. We need to figure out what to do about this first.

**Comment by mikkeloscar** [comment]
The white list is in place for alpha, so we can merge! :)

**Comment by mikkeloscar** [comment]
:+1:

**Pull request merged**